app/models/ability.rb

```diff
@@ -0,0 +1,13 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new # guest user (not logged in)
+    if user.role == 'admin'
+      can :manage, :all # Admin has full access to all resources
```

Comment:
- [ ] [optional] I would suggest using the `admin?` method you created in the user model. It will make your code follow the best practices. 
  - You can consider using this code `if user.admin?`.